templates/pages/setup/general/security_setup.html.twig

```diff
@@ -35,96 +35,146 @@
-</div>
-
-<div class="hr-text">
-    <i class="ti ti-shield-lock"></i>
-    <span>{{ __('Password expiration policy') }}</span>
-</div>
-
-<div class="row ps-4">
-    {{ fields.dropdownNumberField('password_expiration_delay', config('password_expiration_delay'), __('Password expiration delay (in days)'), inline_field_options|merge({
-        'min': 30,
-        'max': 365,
-        'step': 15,
-        'toadd': {'-1': __('Never')}
-    })) }}
-
-    {{ fields.dropdownNumberField('password_expiration_notice', config('password_expiration_notice'), __('Password expiration notice time (in days)'), inline_field_options|merge({
-        'min': 0,
-        'max': 30,
-        'step': 1,
-        'toadd': {'-1': __('Notification disabled')}
-    })) }}
-
-    {{ fields.dropdownNumberField('password_expiration_lock_delay', config('password_expiration_lock_delay'), __('Delay before account deactivation (in days)'), inline_field_options|merge({
-        'min': 0,
-        'max': 30,
-        'step': 1,
-        'toadd': {'-1': __('Do not deactivate')}
-    })) }}
-
-    {{ fields.dropdownTimestampField('password_init_token_delay', config('password_init_token_delay'), __('Validity period of the password initialization token'), inline_field_options|merge({
-        'min': constant('DAY_TIMESTAMP'),
-        'max': constant('MONTH_TIMESTAMP'),
-        'step': constant('DAY_TIMESTAMP')
-    })) }}
-
-    {{ fields.dropdownArrayField(
-        'non_reusable_passwords_count',
-        config('non_reusable_passwords_count'),
-        {
-            '1': __('Last password'),
-            '2': __('Last 2 passwords'),
-            '3': __('Last 3 passwords'),
-            '4': __('Last 4 passwords'),
-            '5': __('Last 5 passwords')
-        },
-        __('Prevent users from reusing previous passwords'),
-        inline_field_options
-    ) }}
-</div>
-
-<div class="hr-text">
-    <i class="ti ti-2fa"></i>
-    <span>{{ __('Two-factor authentication (2FA)') }}</span>
-</div>
-
-<div class="row ps-4">
-    {{ fields.dropdownYesNo('2fa_enforced', config('2fa_enforced'), __('Enforce 2FA'), inline_field_options) }}
-
-    {{ fields.dropdownNumberField('2fa_grace_days', config('2fa_grace_days'), __('2FA grace period (in days)'), inline_field_options|merge({
-        'min': 0,
-        'max': 365,
-    })) }}
-</div>
+   {{ fields.largeTitle(__('Password security policy'), 'ti ti-shield-lock') }}
+
+   {{ fields.smallTitle(__('Security policy validation')) }}
+   {{ fields.sliderField(
+      'use_password_security',
+      config['use_password_security'],
+      __('Force passwords to comply with security policy'),
+      inline_field_options
+   ) }}
+
+   {% if config['use_password_security'] %}
+      {{ fields.smallTitle(__('Security policy configuration')) }}
+
+      {{ fields.numberField(
+         'password_min_length',
+         config('password_min_length'),
```

Comment:
Could you please use `config[...]` instead of `config(...)` int his whole file? We should indead rely on the values injected in the template params rather than the values fetched from the `config()` method, and having both in the same file is quiet confusing.

```suggestion
         config['password_min_length'],
```